app.rb

```diff
@@ -0,0 +1,30 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    'Chitter'
+  end
```

Comment:
redirecting to the place where the peeps are stored might help because most people will access through the '/' route